src/main/java/view/OutputView.java

```diff
@@ -0,0 +1,49 @@
+package view;
+
+import domain.Car;
+import java.util.ArrayList;
+import java.util.List;
+
+public class OutputView {
+
+    public static void printInputCarNames() {
+        System.out.println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).");
+    }
+
+    public static void printInputRoundNumber() {
+        System.out.println("시도할 회수는 몇회인가요?");
+    }
+
+    public static void printResult() {
+        System.out.println("실행 결과");
+    }
+
+    public static void printProcess(List<Car> cars) {
+        for (Car car : cars) {
+            System.out.println(formatCarInfo(car));
+        }
+        System.out.println();
+    }
+
+    public static String formatCarInfo(Car car) {
+        StringBuilder formatted;
+        formatted = new StringBuilder(car.getName() + " : ");
+        for (int i = 0; i < car.getDistance(); i++) {
+            formatted.append("-");
+        }
+        return formatted.toString();
```

Comment:
String의 함수를 잘 활용하면 이렇게도 줄여볼 수 있어요
```suggestion
        return car.getName() + " : " + "-".repeat(car.getDistance());
```